src/pybind/mgr/dashboard/frontend/src/app/core/auth/role-form/role-form.component.html

```diff
@@ -1,75 +1,74 @@
-                   name="description"
-                   formControlName="description">
-          </div>
-        </div>
-
-        <!-- Permissions -->
-        <div class="form-group row">
-          <label i18n
-                 class="cd-col-form-label">Permissions</label>
-          <div class="cd-col-form-input">
-            <cd-checked-table-form [data]="scopes_permissions"
-                                   [columns]="columns"
-                                   [form]="roleForm"
-                                   inputField="scopes_permissions"
-                                   [scopes]="scopes"
-                                   [initialValue]="initialValue"></cd-checked-table-form>
-          </div>
-        </div>
-
+<div cdsCol
+     [columnNumbers]="{md: 4}">
+  <ng-container *cdFormLoading="loading">
+    <form name="roleForm"
+          #formDir="ngForm"
+          [formGroup]="roleForm"
+          novalidate>
+      <div i18n="form title|Example: Create Volume@@formTitle"
+           class="form-header">{{ action | titlecase }} {{ resource | upperFirst }}
       </div>
-      <div class="card-footer">
+        <!--Name -->
+      <div class="form-item">
+        <cds-text-label for="name"
+                        cdRequiredField="Name"
+                        [invalid]="!roleForm.controls.name.valid && roleForm.controls.name.dirty"
+                        [invalidText]="nameError"
+                        i18n>Name
+          <sapn [ngClass]="{'required': mode !== roleFormMode.editing}"></sapn>
+          <input cdsText
+                 placeholder="Name..."
+                 i18n-placeholder
+                 id="name"
+                 name="name"
+                 formControlName="name"
+                 [invalid]="!roleForm.controls.name.valid && roleForm.controls.name.dirty"
+                 autofocus>
+        </cds-text-label>
+        <ng-template #nameError>
+          <span *ngIf="roleForm.showError('name', formDir, 'required')">
+            <ng-container i18n>
+               This field is required.
+            </ng-container>
+          </span>
+          <span *ngIf="roleForm.showError('name', formDir, 'notUnique')">
+            <ng-container i18n>
+                The chosen name is already in use.
+            </ng-container>
+          </span>
+        </ng-template>
+      </div>
+    <!-- Description -->
+      <div class="form-item">
+        <cds-text-label for="name"
+                        i18n>Description
+        <input cdsText
+               type="text"
+               class="input-field"
+               placeholder="Description.."
+               id="description"
+               name="description"
+               formControlName="description">
+        </cds-text-label>
+      </div>
+      <!-- Permissions -->
+      <div  class="form-item">
+        <cds-text-label>Permissions</cds-text-label>
+        <cd-checked-table-form [data]="scopes_permissions"
```

Comment:
I am not able to see the max item count selector on table, is it intended?
![image](https://github.com/user-attachments/assets/c6e2924d-fd9f-4976-a97a-75c40b42f4e9)

Reply:
Did not implemented table  just used the control which present previously need to check